### ISSUE TITLE
What is Nginx being used for?

### ISSUE THREAD

**Issue by HogCat** [fswdevl/Destiny-2-SDR-Matchmaking-Firewall#25]
Was it a previous dependency? Is it for using the same machine to host multiple instances? There doesn't seem to be any references to it beyond installing it, starting it, stopping it, and uninstalling it.

**Comment by ph4br1c3** [comment]
Correct me if I am wrong, but I think it's only to be able to download the openvpn config file "client.ovpn'"

As commented in d2firewall.sh, after 15 min stop nginx and delete openvpn config for security reason.

**Comment by HogCat** [comment]
I would agree, if nginx was configured to serve static file from the home directory, but the default nginx config is just a welcome to nginx screen.  At least as far as I am aware.
